app/channels/room_channel.rb

```diff
@@ -1,7 +1,9 @@
 class RoomChannel < ApplicationCable::Channel
   def subscribed
-    reject if params[:room_id].blank?
+    raise 'No Chat-Id!' if params[:room_id].blank?
```

Comment:
Don't we need here a translation?

Reply:
Not sure is necessary, as it is a system error message (and not a message that a user could see).
But it might be good to add them too.

Reply:
Ok, in that case you could ignore it